scripts/python/proposers_rewards.py

```diff
@@ -242,32 +250,46 @@ def sanity_check_data(
 
 
 def extract_yes_no_votes(proposal: Proposal, voteplan_proposal: ProposalStatus):
+    blank_index = proposal.chain_vote_options["blank"]
     yes_index = proposal.chain_vote_options["yes"]
     no_index = proposal.chain_vote_options["no"]
     # we check before if tally is available, so it should be safe to direct access the data
+    blank_result = voteplan_proposal.tally.results[blank_index]  # type: ignore
     yes_result = voteplan_proposal.tally.results[yes_index]  # type: ignore
     no_result = voteplan_proposal.tally.results[no_index]  # type: ignore
-    return yes_result, no_result
+    return blank_result, yes_result, no_result
 
 
 def calc_approval_threshold(
-    proposal: Proposal, voteplan_proposal: ProposalStatus, threshold: float
+    proposal: Proposal,
+    voteplan_proposal: ProposalStatus,
+    threshold: float,
+    total_stake_threshold: float,
 ) -> Tuple[int, bool]:
-    yes_result, no_result = extract_yes_no_votes(proposal, voteplan_proposal)
+    blank_result, yes_result, no_result = extract_yes_no_votes(
+        proposal, voteplan_proposal
+    )
+    total_stake = blank_result + yes_result + no_result
+    pass_total_threshold = total_stake >= total_stake_threshold
     diff = yes_result - no_result
-    success = diff >= (no_result * threshold)
+    pass_relative_threshold = diff >= (no_result * threshold)
```

Comment:
I think according to the specification it should be `diff > threshold` (the difference between 'yes' and 'no' percentages should be more than the threshold)

Reply:
Lest ask this, because I remember it to be "at least" the threshold.

Reply:
Yes, my mistake. But besides  `> / >=` I actually meant that the calculation is different. The threshold is not on the ratio between 'yes' and 'no' but on the absolute difference between the two

Reply:
Talked with @MarklarIO and it is:
```
IF(B3/C3>=1.15,"YES","NO")

where
B3 is YES votes total per proposal
C3 is NO votes total per proposal
```